servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/CloseUtils.java

```diff
@@ -44,6 +44,9 @@ private CloseUtils() {
      * @param closingStarted a {@link CountDownLatch} to notify
      */
     static void onGracefulClosureStarted(ConnectionContext cc, CountDownLatch closingStarted) {
+        // cc.onClosing() will trigger on the leading edge of closure, which maybe when the user calls closeAsync().
```

Comment:
this was an interesting observation... if we needed to differentiate between "any trigger for closing" (which is what `onClosing` is) and "transport/protocol is closing" we could add another method in the future, but currently this seems to be mostly a concern in tests.